bin/storm.py

```diff
@@ -754,18 +827,24 @@ def parse_config_opts(args):
     elif token == "--config":
       global CONFFILE
       CONFFILE = curr.pop()
+    elif token == "--jars":
+      jars_list.extend(curr.pop().split(','))
+    elif token == "--artifacts":
```

Comment:
No, the term `artifact` is also used from Ivy, too.
http://ant.apache.org/ivy/history/latest-milestone/terminology.html#artifact

I think term `dependencies` is closer to jars + artifacts since jars will be also added to classpath.